Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift

```diff
@@ -15,6 +15,7 @@
 //
 
 import Foundation
+import MatrixSDK
```

Comment:
I'll remove it, XCode auto-imported it, but we don't need it because MatrixSDK is globally imported due to Bridging Header (which might be debatable wether this is a good thing or not)